tests/relay_integration/test_sdk.py

```diff
@@ -53,9 +53,7 @@ def test_recursion_breaker(post_event_with_sdk):
         with pytest.raises(ValueError):
             post_event_with_sdk("internal client test")
 
-    assert_mock_called_once_with_partial(
```

Comment:
I'd find a way to preserve the event ID in the assertion. I think we could have an event_id parameter on the wait_for_ingest_consumer function and inject a hardcoded event ID.

Reply:
On that occasion I would also stop using the RavenShim indirection in that fixture

Reply:
Tried to address both comments.